api/action/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <script defer src="index.js"></script>
+</head>
+<body>
+  <p>Before experimenting with these APIs, we recommend you pin the extension's action button to your
+  toolbar in order to make it easier to see the changes. </p>
+
+  <section id="toggle-state">
+    <h2>enable / disable</h2>
+    <p>Clicking the button will toggle the action button's enabled/disabled state.</p>
```

Comment:
Nit: it wasn't obvious to me that you meant the button right below _on this page_. I tried to click the extension's icon and wondered why it didn't grey out.